src/components/column-padding-control/index.js

```diff
@@ -48,7 +48,7 @@ const ColumnPaddingControl = props => {
 			className="ugb--help-tip-advanced-column-paddings"
 			defaultLocked={ true }
 			min={ [ 0, 0, 0 ] }
-			max={ [ 500, 30, 100 ] }
+			sliderMax={ [ 500, 100, 100 ] }
```

Comment:
Aside from this, are there other low tablet values which are clamped lower than mobile? it may be a good time to increase those if there are any.